ui/yarn.lock

```diff
@@ -17,6 +17,8 @@
   dependencies:
     "@apollo/react-common" "^3.1.3"
```

Comment:
The tests are failing. I wonder if it's because there's something wrong with the `yarn.lock` file. You can try running `yarn` inside the `ui` directory to see if there are any changes.

Reply:
When removing packages, `yarn.lock` usually updates when the package is removed via `yarn remove <package-name>`.